modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockStorage.scala

```diff
@@ -31,7 +31,7 @@ class BlockStorage[F[_]: Sync](blocks: MapRef[F, ProofsHash, Option[StoredBlock]
   def areParentsAccepted(block: DAGBlock): F[Map[BlockReference, Boolean]] =
     block.parent.traverse { ref =>
       isBlockAccepted(ref).map(ref -> _)
-    }.map(_.toNem.toSortedMap)
+    }.map(_.toList.toMap)
```

Comment:
acceptance must be in order, so you need to sort the list at least

Reply:
doesn't it? or it's for something else?

Reply:
It's checking if parents are accepted already - it's used to determine if the block that uses these parents is valid and can be accepted as well.